runtime/lib/app-runtime.js

```diff
@@ -124,7 +124,12 @@ AppRuntime.prototype.init = function init () {
   }
   this.dndMode.init()
   return future.then(() => {
-    if (this.shouldWelcome) {
+    var welcomeAudio = env.welcomeAudio
```

Comment:
So what's the meaning of `env.welcomeAudio`? Does it turn off startup welcoming? Looks like it's `false` by default, which means that do not welcome by default.